.buildconfig-android.yml

```diff
@@ -1,93 +1,93 @@
-libraryVersion: 0.53.1
+libraryVersion: 0.53.2
 groupId: org.mozilla.appservices
 projects:
   fxaclient:
     path: components/fxa-client/android
     artifactId: fxaclient
     publications:
-      - name: fxaclient
-        type: aar
+    - name: fxaclient
+      type: aar
     description: Firefox Accounts client.
   logins:
     path: components/logins/android
     artifactId: logins
     publications:
-      - name: logins
-        type: aar
+    - name: logins
+      type: aar
```

Comment:
Oh, interesting churn in the yaml file here, but I assume it will be a one-time thing.